doc/sedcli.8

```diff
@@ -58,9 +58,17 @@ Activates Locking SP.
 .IP "\fB\-R -d <device> [-i]\fR or \fB\-\-revert --device <device> [--psid]\fR"
 Reverts TPer to Manufactured-Inactivate state using either SID or PSID authority.
 
-.IP "\fB\-L -d <device> -t <locktype>\fR or \fB\-\-lock-unlock --device <device> --locktype {RW|RO|WO|UNLOCK}\fR"
+.IP "\fB\-L -d <device> -t <locktype>\fR or \fB\-\-lock-unlock --device <device> --locktype {RO|RW|LK}\fR"
 Changes the lock state for the device. There is a possibility to setup Read lock,
```

Comment:
Changes the lock state for the device. Following access modes are available: 
1. Read-Only (RO) - user can only read data from the disk.
2. Read-Write (RW) - user can read/write data from/to the disk
3. Locked (LK) - data is locked. It is not possible to read or write data from/to the disk,

Reply:
Please update the manpage to above content

Reply:
Have been changed :)

Reply:
Can you please change description to the suggest description